enhancements/console/dynamic-plugins.md

```diff
@@ -280,10 +358,10 @@ This prevents an uncaught error from causing the application to white-screen
 and break. If a plugin service is unavailable, console will not load the plugin
 and show a message in the notification drawer to let users know.
 
-Additionally, the console can contain a `?disable-plugins` query parameter with
-a comma separated list of plugin names. When this parameter is present, the
-corresponding dynamic plugins are not loaded. If `all` is passed, all dynamic
-plugins will be disabled.
+Additionally, console users can disable specific or all dynamic plugins that
+would normally get loaded by console via `?disable-plugins` query parameter.
+The value of this parameter is either a comma separated list of plugin names
+(disable specific plugins) or an empty string (disable all plugins).
```

Comment:
When we get to the point where everything including the admin perspective is a dynamic plugin, it might not make sense to disable all plugins. But it's useful for now, and I think we can cross that bridge when we come to it.